tool/cli.py

```diff
@@ -100,11 +100,52 @@ def list_branches():
     print(tabulate(a.list_branches(), tablefmt="fancy_grid"))
 
 
+@click.command(name="labels",
+               help="List the labels for the project. "
+                    "This is how you can select a repository for Github: <namespace>/<project>.")
+@click.option('--service', "-s", type=click.STRING, default="github",
+              help="Name of the git service (e.g. github/gitlab).")
+@click.option('--copy', '-c', type=click.STRING, required=False, multiple=True,
+              help="Copy labels to another repo.")
+@click.argument('repo', type=click.STRING, required=False)
+def labels(service, copy, repo):
+    """
+    List the labels for the selected repository, default to repo in $PWD
+    """
+    a = App()
+    if repo:
+        s = a.get_service(service, repo=repo)
+    else:
+        s = a.guess_service()
+    repo_labels = s.list_labels()
+    if not repo_labels:
+        print("No labels requests.")
+        return
+    print(tabulate([
+        (
+            label.name,
+            label.color,
+            label.description
```

Comment:
is it that the library doesn't support this or the api itself?

Reply:
It supports, but the result actually does not contain that...
https://api.github.com/repos/user-cont/tool/labels

Reply:
lol, hopefully ~~github~~ microsoft fixes that sooner or later